src/style.css

```diff
@@ -0,0 +1,43 @@
+
+html {
+  box-sizing: border-box;
+}
+
+* {
+  margin: initial;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__box * {
```

Comment:
Using the universal selector (*) can cause performance issues and unintended style changes. Consider using a specific class for these styles. 🎯
in this case it will be .stars class